apps/workers/sort2file/worker.go

```diff
@@ -107,6 +107,9 @@ func newWorker(info string) task.Worker {
 	// reader(s)
 	stsRdrs := make([]*statsReader, 0)
 	for _, sts := range fSts {
+		if sts.IsDir {
+			continue
+		}
```

Comment:
keep current behaviour by skipping directories

Reply:
same question here, would readers have been created for directories, before this change, or would an error have been thrown due to the directory?